airflow/api_fastapi/core_api/datamodels/tasks.py

```diff
@@ -90,7 +79,7 @@ class TaskResponse(BaseModel):
     @classmethod
     def validate_model(cls, task: Any) -> Any:
         task.__dict__.update(
-            {"class_ref": _get_class_ref(task), "is_mapped": isinstance(task, MappedOperator)}
+            {"class_ref": _get_class_ref(task), "is_mapped": getattr(task, "_is_mapped", False)}
```

Comment:
Having this check multiple times in the code looks like danger of consistency for people making copy&paste - and it is bit made with internal knowledge. Can we move this to a utility such that the check is consistent across the codebase?

Reply:
Good shout. I'll add a property on AbstractOp so we can make this `task.is_mapped` now I think.